README.md

```diff
@@ -14,13 +14,23 @@ The library can be loaded from SQLite as such:
 
 ### Docker
 
-A Docker image has been provided to simplify integrating the library into larger projects. When run, an interactive SQLite session is started with the library pre-loaded for testing.
+A Docker image has been provided to simplify integrating the library into larger projects.
 
-A Docker image with the library pre-built is available on the GitHub Container Registry (soon) to speed up building and simplify integrating into larger projects. If you prefer, you can also build the Docker image yourself:
+- `docker/Dockerfile` builds a minimal image containing just the library. It cannot be run.
+- `docker/demo/Dockerfile` builds the library and starts an interactive SQLite demo session with it loaded.
+
+The library is available as a pre-built Docker image on the GitHub Container Registry (soon) to speed up building and simplify integrating into larger projects. If you prefer, you can also build the Docker image yourself:
```

Comment:
"Both images are provided as packages on the GitHub Container Registry (`/url/library/image` and `/url/demo/image`). If you prefer, you can also build them yourself:"